webauthn-rs-core/src/interface.rs

```diff
@@ -659,6 +663,67 @@ impl AttestationCa {
             strict: false,
         }
     }
+
+    /// Android ROOT CA 1
+    pub fn android_root_ca_1() -> Self {
+        AttestationCa {
+            ca: x509::X509::from_pem(ANDROID_ROOT_CA_1).unwrap(),
+            platform_only: false,
+            key_storage: KeyStorageClass::SingleDeviceWrappedKey,
+            strict: true,
```

Comment:
These shouldn't be in strict, google has been a hostile upstream and has unresolved issues (IE the pixel straight doesn't work)

Reply:
what does the strict here mean?

Reply:
It means the vendor is "high quality", in terms of trust in their CA, responsiveness to issues, crypto quality, hardware quality etc. It's a "project specific" thing, because I've started to find credentials that are "fido certified" that are trash. 

For example, google here can't be strict, because they are a hostile upstream and don't respond to issues. There are defects in the pixels ability to do webauthn at all, and they are "refusing to act on it".

Reply:
Of course, people are still free to use and trust them, but I want to provide a "strict" list which is the "best of the best" of webauthn impl so that not everyone has to do their own investigation into the quality of vendors.